ecs-cli/modules/cli/local/converter/converter.go

```diff
@@ -371,8 +371,7 @@ func convertHealthCheck(healthCheck *ecs.HealthCheck) *composeV3.HealthCheckConf
 		out.Retries = &retries
 	}
 	if healthCheck.StartPeriod != nil {
-		startPeriod := time.Duration(aws.Int64Value(healthCheck.StartPeriod)) * time.Second
-		out.StartPeriod = &startPeriod
+		log.Warn("startPeriod will be ignored and not be supported in the initial release.")
```

Comment:
Can you try to instead changing the Compose version to 3.4 from 3.2 and see if it fixes the issue? 

> Note: start_period is only supported for v3.4 and higher of the compose file format.


https://docs.docker.com/compose/compose-file/#healthcheck

Reply:
Yeah, it works but can we bump to 3.4?

Reply:
According to https://docs.docker.com/compose/compose-file/compose-versioning/, 3.4 is compatible with Docker engine 17.09.0+ which was [released on 2017-09-26](https://docs.docker.com/engine/release-notes/#17090-ce). So it's already been more than a year, I'd say it's fine to bump it 🤔